gateway/src/main/java/org/georchestra/gateway/model/GeorchestraUsers.java

```diff
@@ -25,7 +25,9 @@
 import org.springframework.web.server.ServerWebExchange;
 
 import lombok.NonNull;
+import lombok.experimental.UtilityClass;
 
+@UtilityClass
```

Comment:
As mentioned above, I like it, it tells me what to expect so I can go on instead of trying to figure out what the class does. Oh, it's just a utility class, keep going